client/packages/invoices/src/Prescriptions/DetailView/Toolbar.tsx

```diff
@@ -139,14 +147,9 @@ export const Toolbar: FC = () => {
               Input={
                 <DateTimePickerInput
                   disabled={isDisabled}
-                  defaultValue={defaultPrescriptionDate}
-                  value={DateUtils.getDateOrNull(prescriptionDate)}
```

Comment:
React inputs shouldn't have a "defaultValue" as well as a "value". Defaults are for when the form is being used uncontrolled. In a controlled component, the current value *is* the default.